utils/src/ooinstall/openshift_ansible.py

```diff
@@ -127,8 +127,7 @@ def write_inventory_vars(base_inventory, lb):
             "openshift_master_cluster_public_hostname={}\n".format(lb.public_hostname))
 
     if CFG.settings.get('variant_version', None) == '3.1':
-        # base_inventory.write('openshift_image_tag=v{}\n'.format(CFG.settings.get('variant_version')))
-        base_inventory.write('openshift_image_tag=v{}\n'.format('3.1.1.6'))
+        base_inventory.write('openshift_image_tag=v{}\n'.format(CFG.settings.get('variant_version')))
```

Comment:
Doesn't look quite right, I think you might want to be setting "openshift_release=v3.1", the image tag is meant to be precise.